### ISSUE TITLE
[FEAT] Handling failures

### ISSUE THREAD

**Issue by alexandru-calinoiu** [ElrondNetwork/elrond-eth-bridge#64]
Need to figure out how to handle failure. Batches on eth are atomic, so if a tx in a batch fails all tx's will be reverted and the state should be marked as failed on elrond.

**Comment by alexandru-calinoiu** [comment]
@dorin-iancu how is failure happening on elrond for a batch, can a single tx in a batch fail only? if yes how can the relayers have that info?

**Comment by dorin-iancu** [comment]
Yes, a single tx in a batch can fail, while the rest are successful. In the case of Ethereum -> Elrond tx, the `performAction` endpoint will return the status for each transaction, in the order they were in the proposed batch: `3` for `Executed` and `4` for `Rejected`
